skl2onnx/operator_converters/random_forest.py

```diff
@@ -25,7 +28,11 @@ def _num_estimators(op):
     #       ...
     #       classifier.fit(X_tmp, y_tmp)
     #       classifier.n_estimators += 30
-    return len(op.estimators_)
+    try:
+        return len(op.estimators_)
+    except AttributeError:
+        # HistGradientBoosting
+        return len(op._predictors)
```

Comment:
What if _predictors is not there in op? Let's write it in a way which makes extending it further easier without modification to existing code; something like this:
`if hasattr(op, 'estimators_'):`
`. num_est = len(op.estimators_)`
`elif hasattr(op, '_predictors'):`
`.  num_est = len(op._predictors)`
`else:`
`. # Either raise some error(NotImplementedError) if unexpected or set num_est to 0/None.`
`return num_est`